packages/ses/src/error/tame-console.js

```diff
@@ -1,14 +1,45 @@
 // @ts-check
 
-import { TypeError, globalThis } from '../commons.js';
+import {
+  TypeError,
+  apply,
+  defineProperty,
+  freeze,
+  globalThis,
+} from '../commons.js';
 import { loggedErrorHandler as defaultHandler } from './assert.js';
 import { makeCausalConsole } from './console.js';
 import { makeRejectionHandlers } from './unhandled-rejection.js';
 import './types.js';
 import './internal-types.js';
 
+const wrapLogger = (logger, thisArg) =>
+  freeze((...args) => apply(logger, thisArg, args));
+
 // eslint-disable-next-line no-restricted-globals
-const originalConsole = console;
+const originalConsole = /** @type {typeof console} */ (
+  // eslint-disable-next-line no-nested-ternary
+  typeof console !== 'undefined'
+    ? console
+    : typeof print === 'function'
+    ? // Make a good-enough console for eshost.
+      (p => freeze({ debug: p, log: p, info: p, warn: p, error: p }))(
```

Comment:
Is a console with only these methods good enough for eshost? Is eshost likely to be the only case this code needs to accommodate?